README.md

```diff
@@ -398,7 +402,7 @@ yarn start
 
 </details>
 
-🔄 You know the drill. Restart `yarn chain` and then `yarn deploy` so you can try liquidating on the front end. It may be useful to open a private browser tab and go to `localhost:3000` so you can simulate multiple parties.
+🔄 You know the drill. Run `yarn deploy --reset` so you can try liquidating on the front end. It may be useful to open a private browser tab and go to `localhost:3000` so you can simulate multiple accounts. Notice how the borrower still has their borrowed CORN after they get liquidated. They get to keep their CORN since the liquidator paid their CORN debt back to the protocol on their behalf.
```

Comment:
This might be a good place to add a note that they can use the top right buttons to swap CORN back to ETH.

Reply:
Good plan. Added that note and made the overall instructions clearer here.